src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -14,7 +17,13 @@
 
     Stream<Letter> findByStatus(LetterStatus status);
 
-    Stream<Letter> findByStatusAndSentToPrintAtBefore(LetterStatus status, Timestamp before);
+    @Query("select l from Letter l where l.status = :status and l.type <> '"
+        + UploadLettersTask.SMOKE_TEST_LETTER_TYPE
+        + "' and l.sentToPrintAt < :before")
+    Stream<Letter> findStaleLetters(
+        @Param("status") LetterStatus status,
```

Comment:
either keep the old method name, or remove the `status` parameter.

Perhaps rename and remove param on a future PR?